modules/apps/portal-remote/portal-remote-jaxrs-whiteboard/build.gradle

```diff
@@ -50,4 +50,6 @@ deployDependencies {
 	include "xmlschema-core-*.jar"
 
 	rename(/geronimo-jaxrs_2\.1_spec-(.+)\.jar/, "geronimo-jaxrs_2.1_spec" + renameSuffix)
+	rename(/geronimo-json_1\.1_spec-(.+)\.jar/, "geronimo-json_1.1_spec" + renameSuffix)
```

Comment:
@martamedio Why you add this and remove it in next commit? Could you please merge the two commits into one?

Thanks
Tina